client/components/dashboard/AddItem.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import MenuItem from '@mui/material/MenuItem';
+import { InputLabel, Select } from '@mui/material';
+import OutlinedInput from '@mui/material/OutlinedInput';
+import FormControl from '@mui/material/FormControl';
+
+const theme = createTheme();
+
+const categories = ['Childcare', 'Clothing', 'Education', 'Food', 'Healthcare', 'Homegoods', 'Personal hygiene', 'Other'];
+
+export default function Add() {
+  const [inputs, setInputs] = useState({
+    item: '',
+    category: '',
+    quantity: 0,
+  });
+
+  const handleChange = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    setInputs(values => ({...values, [name]: value}));
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const data = new FormData(document.getElementById('addItem'));
+    console.log('item', data.get('item'), 'category', data.get('category'), 'quantity', data.get('quantity'));
+    fetch('/api/items', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(data),
+    })
+      .then(res => console.log(res));
+
+    setInputs({
+      item: '',
+      category: '',
+      quantity: 0,
+    });
+  };
+
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
I haven't looked too much into Material UI themes but is there a purpose to setting up another ThemeProvider her when pages are given a ThemeProvider via the Layout template. It seems like we should be able to provide it once and be set. Similar comment for CssBaseline and the Dashboard component